Resources/Prototypes/_Sunrise/Entities/Mobs/Player/ghost_roles.yml

```
@@ -329,7 +329,7 @@
     pocket1: Holoprojector
     pocket2: HoloprojectorField
   inhand:
-  - WeaponEnergyGun
+  - WeaponLauncherHydra
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)